html/janus.js

```diff
@@ -3164,14 +3121,13 @@ function Janus(gatewayCallbacks) {
 			config.bitrate.value = null;
 			try {
 				// Try a MediaStreamTrack.stop() for each track
-				if(!config.streamExternal && config.myStream !== null && config.myStream !== undefined) {
+				if(!config.streamExternal && config.myStream) {
 					Janus.log("Stopping local stream tracks");
 					var tracks = config.myStream.getTracks();
 					for(var i in tracks) {
 						var mst = tracks[i];
 						Janus.log(mst);
-						if(mst !== null && mst !== undefined)
-							mst.stop();
+						if(mst) mst.stop();
```

Comment:
Please don't collapse condition and action. As per our code style, they're always on different lines.